enumerable_methods.rb

```diff
@@ -173,3 +184,7 @@ def my_map(prc)
 end
 
 # rubocop: enable Metrics/ModuleLength
```

Comment:
## my_map
**[REQUIRED]**

* Returns a new array with the results of running block once for every element in enum.
```
Failure/Error: expect(array.my_map(&block)).to eq array.map(&block)
      
      ArgumentError:
        wrong number of arguments (given 0, expected 1)
```
* Returns an Enumerator if no block is given
```
Failure/Error: expect(array.my_map).to be_an(Enumerator)
      
      ArgumentError:
        wrong number of arguments (given 0, expected 1)
```